<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgApOne</class>
 <widget class="QDialog" name="DlgApOne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Поточна стадія</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_State"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="bnStart">
           <property name="text">
            <string>Розпочати набір</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bnConcretion">
           <property name="text">
            <string>Згущення</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bnRarefaction">
           <property name="text">
            <string>Розкачка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bnGrowth_1">
           <property name="text">
            <string>Ріст 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bnGrowth_2">
           <property name="text">
            <string>Ріст 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bnGrowth_3">
           <property name="text">
            <string>Ріст 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bnClasp">
           <property name="text">
            <string>Уварювання</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Вибір продукту</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="bx_prod">
        <item>
         <property name="text">
          <string>Сироп</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Сироп+БП</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Біла патока</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Вода</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>56</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Корекція</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QDial" name="dl_Kor">
           <property name="minimum">
            <number>-25</number>
           </property>
           <property name="maximum">
            <number>25</number>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="notchesVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="sb_Kor">
           <property name="minimum">
            <number>-25</number>
           </property>
           <property name="maximum">
            <number>25</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bnStop">
        <property name="text">
         <string>Зупинити</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="bnSetup">
     <property name="text">
      <string>Параметри</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>127</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="bnExit">
     <property name="text">
      <string>Закрити</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="apMnemo">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>421</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>352</width>
       <height>422</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bnExit</sender>
   <signal>clicked()</signal>
   <receiver>DlgApOne</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
